package NSocial;

enum ESocialNetwork {
    UnknownNetwork = 0;
    YaRu = 1;
    Facebook = 2;
    Twitter = 3;
    VKontakte = 4;
    Odnoklassniki = 5;
    LiveJournal = 6;
    MoiMirMailRu = 7;
    LinkedIn = 8;
    MoiKrug = 9;
    GooglePlus = 10;
    DiaryRu = 11;
    LiveInternet = 12;
    FreeLance = 13;
    FriendFeed = 14;
    MirTesen = 15;
    Beon = 16;
    Foursquare = 17;
    Instagram = 18;
    Github = 19;
    NETWORKS_COUNT = 20; // you must update this value when adding a new network
}
